### DISCUSSION TITLE
question.  how to share  recipe among groups

### DISCUSSION THREAD

**Discussion by sdetweil** [mealie-recipes/mealie#3301] [General]
my adult sisters w families and are sharing an instance, w different  groups

now I want to share  a recipe w them.

they get the link as I sent it, it opens to view the recipe
but they cannot save it, to have it in their recipe list.

user can be in only one group...

**Comment by michael-genson** [comment]
Groups are effectively isolated instances; you can think of them as completely different sites. Thus, while you can send links from one group to a person in another group, they can't interact with them (other than reading them).

There is some discussion around a "Households" feature which more tightly integrates sharing of recipes between different households, though no work has been done on it as far as I know, outside of discussion. More on that here: https://github.com/mealie-recipes/mealie/discussions/2134
